README.md

```diff
@@ -1,2 +1,28 @@
 # MH4U-Dex
 Joseph Goldberg's apprentice project. 
+
+Open-Source under MIT License: http://opensource.org/licenses/MIT
```

Comment:
Consider linking to the `LICENSE` file in addition to or instead of the generic license at this URL.